Formula/qpdf.rb

```diff
@@ -16,6 +16,7 @@ class Qpdf < Formula
   end
 
   depends_on "jpeg"
+  depends_on "openssl@1.1"
```

Comment:
Is this not compatible with OpenSSL 3?

Reply:
It is, but then it conflicts with ocrmypdf, that's why I reverted back to 1.1.